src/snapred/backend/recipe/algorithm/CustomGroupWorkspace.py

```diff
@@ -24,19 +34,31 @@ def PyExec(self):
         self.getProperty("InstrumentName").value
         outputWorkspace = self.getProperty("OutputWorkspace").value
 
-        CreateWorkspace(OutputWorkspace="idf", DataX=1, DataY=1)
-        LoadInstrument(
-            Workspace="idf",
-            Filename="/SNS/SNAP/shared/Calibration/Powder/SNAPLite.xml",
-            MonitorList="-2--1",
-            RewriteSpectraMap=False,
-        )
+        donorWorkspace = self.getProperty("InputWorkspace").value
+        loadEmptyInstrument = bool(not donorWorkspace)
+        if loadEmptyInstrument:
+            donorWorkspace = "idf"  # name doesn't matter
+            CreateWorkspace(OutputWorkspace=donorWorkspace, DataX=1, DataY=1)
+            LoadInstrument(
+                Workspace=donorWorkspace,
+                Filename="/SNS/SNAP/shared/Calibration/Powder/SNAPLite.xml",
+                MonitorList="-2--1",
```

Comment:
I think we concluded earlier that `MonitorList` doesn't do anything as input and that it's unclear what "-2--1" means.

Reply:
I think it means pixels -2 to -1 are monitors.  The SNAPLite definition has two monitors in it.

Reply:
Do we want to always and only ever load the SNAPLite instrument?

Reply:
> I think it means pixels -2 to -1 are monitors.

OK, maybe in the old days that property was used to set up monitor pixels. But now `MonitorList` is an _output_ property.  The C++ implementation touches it only at the end of `LoadInstrument::exec()`:
```
// Set the monitors output property
setProperty("MonitorList", (ws->getInstrument())->getMonitors());
```
In other words, I don't see how us passing that list as input would affect things.

Reply:
Removing it is probably a good idea, since this might presumably be used on non-SNAP instruments at some point

Reply:
Related: we have a ton of places where the instrument is hard-coded as "SNAP" even though there is an abstraction for instrument in `Config`. That refactor should be a separate PR.